README.md

```diff
@@ -47,9 +62,9 @@ To get these examples to compile, you will need to install
 The last line could be put in your `~/.bash_profile` file so that you don't have to type each time you run the LF program in a new shell.
 
 To run the compiled code, you need an MQTT broker to be running. For example, the [Mosquitto Eclipse project](https://mosquitto.org/download/) provides one. On a Mac, you can use homebrew to install the Mosquitto broker:
-
+```shell
     brew install mosquitto
-
+```
 To start the broker and test it, do this:
```

Comment:
I think we need a new section header here.  All Platforms?

Reply:
Yes, good point thanks